cydarm.json

```diff
@@ -0,0 +1,1988 @@
+            "verbose": "Assumes a mimeType='text/plain' and a significance='Comment'."
+        },
+        {
+            "action": "create case comment",
+            "identifier": "create_case_data_comment",
+            "description": "Create a plaintext comment on a case",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "case_uuid": {
+                    "name": "case_uuid",
+                    "description": "UUID of the case",
+                    "order": 0,
+                    "data_type": "string",
+                    "required": true
+                },
+                "data": {
+                    "name": "data",
+                    "description": "Comment text",
+                    "order": 1,
+                    "data_type": "string",
+                    "required": true
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.case_uuid",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.parameter.data",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.acl",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.uuid",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "create case",
+            "identifier": "create_case",
+            "description": "Create case",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "acl": {
+                    "name": "acl",
+                    "description": "The ACL of the data object",
+                    "order": 0,
+                    "data_type": "string"
+                },
+                "assignee": {
```

Comment:
Is this like a username, full name, first name only, a role, or email?

Reply:
Changed description to specify email